lib/fine-types/test/Language/FineTypes/Export/OpenAPI/TypSpec.hs

```diff
@@ -0,0 +1,42 @@
+module Language.FineTypes.Export.OpenAPI.TypSpec
+    ( spec
+    ) where
+
+import Prelude
+
+import Control.DeepSeq
```

Comment:
why we need to summon the evil here? deepseq is a misfeature,  isn't Eq enough to prove equality ?

Reply:
Ah, I wanted to make explicit that the generated schema does not contain `error`. 🤔 When reading

```
schema `shouldBe` schema
```

one might be inclined to think "That should always be true, why is this even a test?!", whereas

```
force schema `shouldBe` schema
```

makes it more clear that the test is about checking for the presence of `error`. Happy to formulate it in a different way, though.

Reply:
I have made this intention of mine more explicit by formulating the test as

```hs
        it "works, i.e. does not contain ⊥" $ do
            m <- readModule
            let schema = schemaFromModule m
            hasNormalForm schema

hasNormalForm :: NFData a => a -> Expectation
hasNormalForm x = rnf x `shouldBe` ()